packages/babel-plugin-ember-test-metadata/utils.js

```diff
@@ -56,14 +56,40 @@ function getProjectConfiguration(project) {
   return parsedProjectConfiguration;
 }
 
+function _addonFound(pathSegments, projectConfiguration) {
+  const addonPaths = projectConfiguration.pkg['ember-addon'].paths;
+
+  if (!addonPaths) {
+    return;
+  }
+
+  const addonNames = addonPaths.map((path) => path.replace('lib/', ''));
+
+  return pathSegments.filter((addonName) => addonNames.includes(addonName))[0];
```

Comment:
Should we be using filter here? We're trying to find the addon name from the pathSegments, right? Should we not instead use `find`?